membersrvc/ca/ca.go

```diff
@@ -576,8 +581,8 @@ func (ca *CA) validateAndGenerateEnrollID(id, affiliation string, role pb.Role)
 func (ca *CA) registerUser(id, affiliation string, role pb.Role, registrar, memberMetadata string, opt ...string) (string, error) {
 	memberMetadata = removeQuotes(memberMetadata)
 	roleStr, _ := MemberRoleToString(role)
-	Trace.Printf("Received request to register user with id: %s, affiliation: %s, role: %s, registrar: %s, memberMetadata: %s\n",
-		id, affiliation, roleStr, registrar, memberMetadata)
+	caLogger.Debugf("Received request to register user with id: %s, affiliation: %s, role: %s, registrar: %s, memberMetadata: %s\n",
+		id, affiliation, affiliationRole, roleStr, registrar, memberMetadata)
```

Comment:
FYI the tests failed here because affiliationRole isn't defined (I assume you meant to rename roleStr)